bundles/org.openhab.binding.radiothermostat/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -20,6 +18,30 @@
 				<type>radiothermostat:next_time</type>
 			</add-channel>
 		</instruction-set>
+
+		<instruction-set targetVersion="3">
+			<update-channel id="temperature">
```

Comment:
The instruction should have a remove for `temp-temperature` and an add for`temperature`.
Same applies to other channels that are renamed. Only channels that hold their id's are properly processed by `update`.

Reply:
The channel ids are not renamed, only the channel type ids. So I believe `update-channel` is correct.

Reply:
Thanks @jlaur for correcting me here. I mixed the two.